### PR TITLE
feat(json): add json output

### PR DESCRIPTION
**Pull request by MrCloudSec**
### Context 

Add feature to create Json output.

### Description

Include JSON model with Pydantic and functionalities to create output Json files.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: lib/outputs/outputs.py [hunk 1]
```diff
@@ -51,6 +53,13 @@ def report(check_findings, output_options, audit_info):
                 )
                 csv_writer.writerow(finding_output.__dict__)
 
+            if "json" in file_descriptors:
+                finding_output = Check_Output_JSON(**finding.check_metadata.dict())
```
Comment (n4ch04) [inline lib/outputs/outputs.py:57]: Check this, do we really need `**` ? And if so, how it works underneath?

File: lib/outputs/outputs.py [hunk 2]
```diff
@@ -51,6 +53,13 @@ def report(check_findings, output_options, audit_info):
...
+                fill_json(finding_output, audit_info, finding)
```
Comment (n4ch04) [inline lib/outputs/outputs.py:58]: Check how to fill object inherited from pydantic

File: lib/outputs/outputs.py [hunk 3]
```diff
@@ -104,3 +130,32 @@ def generate_csv_fields():
...
+    return finding_output
+
+
+def close_json(output_directory, audited_account):
+    filename = f"{output_directory}/prowler-output-{audited_account}-{json_file_suffix}"
+    file_descriptor = open_file(
+        filename,
+        "a",
+    )
+    file_descriptor.seek(file_descriptor.tell() - 1, os.SEEK_SET)
```
Comment (n4ch04) [inline lib/outputs/outputs.py:158]: Please add details here

**Comment by n4ch04** [comment]
Maaaaan, nice job but you know you have to include tests

**Review by jfagoagas — APPROVED** [review]
Good job @sergargar 🚀

**Pull request merged**